pringo.opam

```diff
@@ -2,18 +2,29 @@ opam-version: "2.0"
 name: "pringo"
 synopsis: "Pseudo-random, splittable number generators"
 description:
-  "Pseudo-random number generators that support splitting and two interfaces: one stateful, one purely functional"
+  "Pseudo-random number generators that support splitting and two interfaces: one stateful, one purely functional."
```

Comment:
I know `opam lint` complains about it but to reduce the pain updating to newer releases I would suggest leaving it as-is.

Reply:
Done!